persistence/ravendb/index.md

```diff
@@ -27,19 +27,16 @@ partial: glance
 
 Specific versions of RavenDB Persistence are tied to a major version of NServiceBus and also designed to work with a specific version of the RavenDB client library. When releasing a new major version of NServiceBus, the corresponding RavenDB Persistence release will use the last supported version of RavenDB, so that it is never necessary to upgrade both NServiceBus and RavenDB at the same time.
 
-| NServiceBus | RavenDB Persistence | RavenDB Client | Platform    |
-|:-----------:|:-------------------:|:--------------:|:-----------:|
-|     7.x     |        6.x          |       4.2      | .NET 4.7.2 / .NET Core 2.1  |
-|     7.x     |        5.0.x        |       3.5      | .NET 4.5.2 / .NET Core 2.0  |
-|     6.x     |        4.2.x        |       3.5      | .NET 4.5.2  |
-|     6.x     |        4.0.x        |       3.0      | .NET 4.5.2  |
-|     5.x     |        3.2.x        |       3.5      | .NET 4.5.2  |
-|     5.x     |        3.0.x        |       3.0      | .NET 4.5.2  |
-|     5.x     |         2.x         |       2.5      | .NET 4.5.2  |
+| RavenDB Persistence | RavenDB Client |
+|:-------------------:|:--------------:|
+|        7.x          |       5.2      |
```

Comment:
Since the upgrade guide for v8 is already visible we could also add it already maybe, but I was on the fence since until then, we might already support RavenDB client 27;)?